_posts/jbfelder/2017-05-22-jbfelder-turtle-hack.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+author: tommytester
+title: "jbfelder's Turtle Exercise"
+---
+
+Here's the program I'm embedding:
+  <iframe src="https://trinket.io/embed/python/13abbf94fc" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+
+  While using Turtle is rather basic, having to create the perfect combination or sequences of code were exceedingly difficult. At first, I assumed that I knew what I was doing and proceeded to write all the code as if I knew what was going to happen. Doing that created many problems as I couldn't find where problems exactly occured. To solve this I went and commented out each line and uncommented it out one by one until I could pinpoint what went wrong. Then, I would comment everything out again and repeat the process to make sure that none of the previous lines were also affected. After finding what went wrong, not much extra time was needed to determine how to get the program to work.
```

Comment:
Your reflection doesn't have any specifics about your program and I can't view your turtle hack because it looks like you embedded the wrong one.

As you work on new turtle programs, make sure to create new ones or duplicate them so that you don't clobber your old work.  Please get this post updated so that I can review it.  And make sure that you don't update your Turtlehack program -just embed it as it was as of this post.